addons/project/project.py

```diff
@@ -162,9 +162,9 @@ def _get_favorite(self, cr, uid, ids, name, args, context=None):
 
     def _set_favorite(self, cr, uid, ids, field_name, field_value, args, context=None):
         if field_value:
-            self.write(cr, uid, ids, {'favorite_user_ids': [(4, uid)]}, context=context)
+            self.write(cr, SUPERUSER_ID, ids, {'favorite_user_ids': [(4, uid)]}, context=context)
         else:
-            self.write(cr, uid, ids, {'favorite_user_ids': [(3, uid)]}, context=context)
+            self.write(cr, SUPERUSER_ID, ids, {'favorite_user_ids': [(3, uid)]}, context=context)
```

Comment:
@atp-odoo 
Your commit is the good one but I think it should be merged in saas-10 instead.